mgz/header/initial.py

```diff
@@ -61,14 +61,16 @@
     Embedded(IfThenElse(lambda ctx: ctx._.restore_time == 0,
         Struct(
             "objects"/RepeatUpTo(b'\x00', existing_object),
-            Const(b'\x00\x0b'),
+            Find(b'\x00\x0b', None), # Some additional data was added in 63.0, so the separator is not aligned anymore. Since we don't know what this data represents, we skip it.
```

Comment:
This will result in incorrectly parsing the objects. The separator is almost certainly still in place.

Reply:
The separator is in place, but 3000-4000 bytes later than expected. The find simply skips the data in-between

Reply:
It's later than expected because the preceding objects are not being parsed correctly, right?

Reply:
I dug deeper and was able to revert this part by correcting the parsing of combat actions. Something changed in case the AI is of type `15` or `17`, whatever that means. I also moved around a byte to make some variables align correctly and get meaningful values.
This seems to be working for current files, but the tests seem to not work for older versions of the recs. I'll have to investigate further why that is. I probably forgot a conditional somewhere.